### PR TITLE
Added system parameter for disabling iterating map stats

### PR DESCRIPTION
**Pull request by muratayan**

### CODE REVIEW

**Comment by emrahkocaman** [comment]
:+1:

**Comment by serkan-ozal** [comment]
:+1:

**Comment by Serdaro** [comment]
Needs an update in the Reference Manual?

**Comment by muratayan** [comment]
@Serdaro yes, I'll update if this gets merged.

File: hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java [hunk 1]
```diff
@@ -293,6 +293,8 @@
     ENABLE_JMX_DETAILED("hazelcast.jmx.detailed", false),
     JMX_UPDATE_INTERVAL_SECONDS("hazelcast.jmx.update.interval.seconds", 5, SECONDS),
 
+    DISABLE_ITERATING_MAP_STATS("hazelcast.management.disable.iterating.map.stats", false),
```
Comment (bilalyasar) [inline hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java]: our all `enabling/disabling` property names end with `X.X.X.enabled`, why you don't use that formation?

**Pull request merged**